WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -241,6 +241,9 @@ private extension OrdersViewController {
         tableView.backgroundColor = .listBackground
         tableView.refreshControl = refreshControl
         tableView.tableFooterView = footerSpinnerView
+        tableView.estimatedSectionHeaderHeight = Settings.estimatedHeaderHeight
+        tableView.sectionHeaderHeight = UITableView.automaticDimension
```

Comment:
just curious, do you know if there are any differences setting the estimated section header height & section header height here vs. in `UITableViewDelegate` implementation like in the deleted diffs?

Reply:
Based on my short experience, no difference. 😄 Except that the delegate method allows you to set different heights for different sections.